pipemgr/src/templates/job.yml

```diff
@@ -14,7 +14,7 @@ plan:
           username: ((ce-dockerhub-id))
           password: ((ce-dockerhub-token))
           repository: openstax/content-synchronizer
-          tag: 20211027.213953
+          semver_constraint: ">= 20211213.17322"
```

Comment:
This comment is probably moot because content-synchronizer is just meant to be short-lived but if we need to debug a failure in concourse will we know which version of content-synchronizer was running at the time?

Reply:
I did not think about that. Yes, you can find that information in the build log. Inside the image fetching details for the sync-content task, you should be able to see the tag and sha for the image used. Example: osbooks-history's build 2 used 20211027.213953.

Ideally, the tag and sha for both images should always match because they are automatically generated at the same time from the same source code.